packages/contracts/contracts/L1/ValidatorManager.sol

```diff
@@ -600,6 +596,15 @@ contract ValidatorManager is ISemver, IValidatorManager {
                 COMMISSION_RATE_DENOM - commissionRate,
                 COMMISSION_RATE_DENOM
             );
+
+            uint128 validatorKro = ASSET_MANAGER.totalValidatorKro(validator);
+            uint128 totalKro = ASSET_MANAGER.totalKroAssets(validator);
+            uint128 validatorBaseReward = baseReward.mulDiv(validatorKro, totalKro + validatorKro);
+            // Exclude the base reward for the validator from total base reward given to KRO delegators.
+            baseReward -= validatorBaseReward;
+            validatorReward =
+                (BASE_REWARD + boostedReward).mulDiv(commissionRate, COMMISSION_RATE_DENOM) +
```

Comment:
boostedReward is already decreased above. Need to calculate the commission from boosted reward before decreased.